README.md

```diff
@@ -168,7 +168,7 @@ foreach (var service in shardStatuses.Services)
 You can find a list of all the available operations in [StatusRiotApi in the documentation](http://benfradet.github.io/RiotSharp/api/RiotSharp.StatusRiotApi.html).
 
 
-For a full description check the [RiotSharpExample](RiotSharpExample) or [RiotSharpTest](RiotSharpTest) projects.
+For a full description check the [RiotSharpTest](RiotSharpTest) project.
```

Comment:
I don't think the unit test project is a usefull description for new users. You might want to remove this line.

Reply:
I removed the example project so this is what's left, either that and/or the `IRiotApi` interface.

Why did I remove the example project? It presents only a small subset of the api and I don't think it's that useful unfortunately.